backend/functions/lib/testing/mock-data/cams-http-request-helper.ts

```diff
@@ -7,7 +8,9 @@ export function mockCamsHttpRequest(override: Partial<CamsHttpRequest> = {}): Ca
     query: {},
     method: 'GET',
     url: mockRequestUrl,
-    headers: {},
+    headers: {
+      authorization: 'Bearer ' + MOCK_AUTHORIZATION_BEARER_TOKEN,
```

Comment:
We could replace all the instances of this `MOCK_AUTHORIZATION_BEARER_TOKEN` with `MockData.getJwt`.